Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/DirectILLCollectData.py

```diff
@@ -200,32 +199,6 @@ def _monitorCounts(ws):
         return logs.getProperty("monitor.monsum").value
 
 
-def _get_instrument_structure(ws):
```

Comment:
what was the reason for removing this?

Reply:
This function had an "undefined variable" as well. When I did a grep for this function name, its not being used anywhere else in the codebase. Also its in the `plugins/algorithms/WorkflowAlgorithms` directory so I believe its not used in any user scripts